extensions/cl_mobileye_reservation_sets.asciidoc

```diff
@@ -0,0 +1,545 @@
+    by the OpenCL implementation on the device.
+  * `CL_OUT_OF_HOST_MEMORY` if there is a failure to allocate resources
+    required by the OpenCL implementation on the host.
+
+After the _reservation_set_ reference count becomes zero and the command-buffer
+associated with _reservation_set_ is not executing, the reservation-set object
+is deleted.
+--
+
+[open,refpage='clCommandNDRangeKernelReservationSetMOBILEYE',desc='Record an ND-range kernel command into a reservation-set.',type='protos']
+--
+To record an ND-range kernel command into a reservation-set, call the function:
+include::{generated}/api/protos/clCommandNDRangeKernelReservationSetMOBILEYE.txt[]
+include::{generated}/api/version-notes/clCommandNDRangeKernelReservationSetMOBILEYE.asciidoc[]
+
+  * _reservation_set_ is a valid reservation-set object.
+  * _command_queue_ specifies the command-queue the command will be recorded to.
+    If the `cl_khr_command_buffer_multi_device_EXT` extension is not supported,
+    only a single command-queue is supported, and _command_queue_ must be
+    `NULL`. If the `cl_khr_command_buffer_multi_device_EXT` extension is
+    supported and _command_queue_ is `NULL`, then only one command-queue must
+    have been set on _command_buffer_ creation; otherwise, _command_queue_ must
+    not be `NULL`.
+  * _properties_ specifies a list of properties for the kernel command and their
+    corresponding values. Each property name is immediately followed by the
+    corresponding desired value. The list is terminated with 0. If a supported
+    property and its value is not specified in _properties_, its default value
+    will be used. _properties_ may be `NULL`, in which case the default values
+    for supported properties will be used.
+  * _kernel_ is a valid kernel object.
+  * _work_dim_, _global_work_offset_, _global_work_size_, _local_work_size_
+    Refer to *clEnqueueNDRangeKernel*.
+  * _sync_point_ returns a synchronization-point ID that identifies this
+    particular command. Synchronization-point objects are unique and can be used
+    to identify this command later on. _sync_point_ can be `NULL` in which case
+    it will not be possible for the application to record a wait for this
+    command to complete.
+  * _mutable_handle returns a handle to the command. If the
+    `cl_khr_command_buffer_mutable_dispatch` extension is supported, and
+    _mutable_handle_ is not `NULL`, it can be used in the
+    `cl_mutable_dispatch_config_khr` struct to update the command configuration
+    between recordings. The lifetime of this handle is tied to the parent
+    command-buffer, such that freeing the command-buffer will also free this
+    handle.
+
+*clCommandNDRangeKernelReservationSetMOBILEYE* returns `CL_SUCCESS` if the
+function is executed successfully. Otherwise, it returns the errors defined by
+*clEnqueueNDRangeKernel* except::
+
+  * `CL_INVALID_RESERVATION_SET_MOBILEYE` if _reservation_set_ is not a valid
```

Comment:
You should grab a new enum value for this in the xml https://github.com/KhronosGroup/OpenCL-Docs/blob/696e82027d2baa650c894fb1b2524afef717f52e/xml/cl.xml#L788